app/src/main/java/com/example/vybe/MapFragment.java

```diff
@@ -170,53 +168,47 @@ public void onLowMemory() {
         mMapView.onLowMemory();
     }
 
+    public void setToLocation(LatLng latLng) {
+        Log.d(TAG, "setToLocation: Here");
+        mMap.clear();
+        addMarker(latLng, R.drawable.ic_map_marker);
+        setCamera(latLng);
+    }
+
+    public void setToCurrentLocation() {
+        Location location = LocationController.getUserLocation(getContext());
+        if (location == null) {
+            return;
+        }
+        setCamera(new LatLng(location.getLatitude(), location.getLongitude()));
+    }
+
+    public void addMarker(LatLng latLng, @DrawableRes int drawableRes) {
+        BitmapDescriptor marker = vectorToBitmap(drawableRes);
+        mMap.addMarker(new MarkerOptions().position(latLng).icon(marker));
+    }
+
+    private void setCamera(LatLng latLng) {
+        CameraUpdate cameraUpdate = CameraUpdateFactory.newLatLngZoom(latLng, 15);
```

Comment:
i think the 15 here (and any other hard coded values) can be moved to the constants file but ok

Reply:
Aii